server/sonar-web/src/main/js/apps/issues/components/App.js

```diff
@@ -572,6 +599,19 @@ export default class App extends React.PureComponent {
     });
   };
 
+  handlePopupToggle = (issue /*: string */, popupName /*: string */, open /*: ?boolean */) => {
+    this.setState((state /*: State */) => {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)